syntax = "proto3";

package adder;

// The adder service definition.
service Adder {
  // Sends two numbers and receives their sum.
  rpc AddNumbers (AddRequest) returns (AddReply) {}
}

// The request message containing two numbers.
message AddRequest {
  int32 number1 = 1;
  int32 number2 = 2;
}

// The response message containing the sum.
message AddReply {
  int32 sum = 1;
}
